draft-ietf-ppm-dap.md

```diff
@@ -1542,15 +1568,27 @@ struct {
 } PrepareResp;
 ~~~
 
-First the Helper preprocesses each report as follows:
+To begin, the Helper MAY resolve replays with the following procedure. It MUST
+perform this procedure prior to completing the aggregation job. For each report
```

Comment:
It's a bit confusing that this is a MAY in one sentence and a MUST in the next. I guess the idea is that the helper is allowed to defer replay checks if it wants to?

Reply:
Yes, that's the idea.

Reply:
Alright. This a little more laborious but I think it's less ambiguous:

> The Helper MUST resolve replays with the following procedure. It MAY do so immediately upon receiving the `AggregationJobInitReq`, but it MUST do so prior to completing the aggregation job. For each report...

Reply:
Done.